### ISSUE TITLE
[REGRESSION] Firefox keyboard doesn't type any more

### ISSUE THREAD

**Issue by jperl** [microsoft/playwright#1009]
**Context:**
- Playwright Version: 0.11.0
- Operating System: Mac
- Extra: Works on 0.10.0

**Code Snippet**

```javascript
const playwright = require("playwright");

(async () => {
  for (let browserType of ["chromium", "firefox"]) {
    const browser = await playwright[browserType].launch({ headless: false });
    const page = await browser.newPage();
    await page.goto("https://google.com");
    await page.type('css=[aria-label="Search"]', "Hello");
    await page.keyboard.press("Enter");
  }
})();
```

**Describe the bug**

Enter does not work on Firefox but does on Chromium. Issue is for press and up/down.

**Comment by pavelfeldman** [comment]
Thanks for the heads up. I think it regressed in https://github.com/microsoft/playwright/pull/983, relavant code was touched here: https://github.com/dgozman/gecko-dev/commit/7faa4dd53e79032090a7718dc6dd801c3180f7cf#diff-f489daebcea1150bf9b5d3015347cadfR678

@dgozman ^^

**Comment by aslushnikov** [comment]
Should be fixed with #1023!
